showcases/react-showcase/src/components/card/index.tsx

```diff
@@ -10,25 +10,33 @@ const getCard = ({
 	children,
 	spacing,
 	elevationLevel
-}: DBCardProps) => (
-	<DBCard
-		behavior={behavior}
-		spacing={spacing}
-		elevationLevel={elevationLevel}>
-		<strong>{children}</strong>
-	</DBCard>
-);
+}: DBCardProps) => {
+	const card = (
+		<DBCard
+			behavior={behavior}
+			spacing={spacing}
+			elevationLevel={elevationLevel}>
+			<strong>{children}</strong>
+		</DBCard>
+	);
 
-const CardComponent = (props: BaseComponentProps) => {
-	return (
-		<DefaultComponent
-			title={'DBCard'}
-			variants={getVariants(
-				defaultComponentVariants,
-				getCard,
-				props.slotCode
-			)}></DefaultComponent>
+	return behavior === 'interactive' ? (
+		<button type="button" aria-label="button copy">
```

Comment:
```suggestion
		<button type="button">
```

Probably this aspect is too complex to explain, so we should remove this part.